### PR TITLE
Added delete application plan method to gem along with the rspec test

### PR DESCRIPTION
**Pull request by brendenowens**

### CODE REVIEW

File: lib/3scale/api/client.rb [hunk 1]
```diff
@@ -242,6 +242,16 @@ def create_application_plan(service_id, attributes)
         extract(entity: 'application_plan', from: response)
       end
 
+
+      # @api public
+      # @return [Bool]
+      # @param [Fixnum] service_id Service ID
+      # @param [Fixnum] application_plan_id Application Plan ID
+      def delete_application_plan(service_id,application_plan_id)
+        response = http_client.delete("/admin/api/services/#{service_id}/application_plans/#{application_plan_id}")
```
Comment (eguzki) [inline lib/3scale/api/client.rb]: `response` is not used

Comment (brendenowens) [reply lib/3scale/api/client.rb]: Removed the response variable from method.

**Comment by eguzki** [comment]
@brendenowens the PR conflicts after merging another PR https://github.com/3scale/3scale-api-ruby/pull/42

Can you please rebase it?

**Comment by brendenowens** [comment]
Rebase done

**Comment by eguzki** [comment]
sorry, something is wrong.

The PR should consist of just two commits https://github.com/3scale/3scale-api-ruby/pull/43/commits/9aa682503f097afce569b2fe49b911cc1b521986 and https://github.com/3scale/3scale-api-ruby/pull/43/commits/638a201761330876e90aa13d8f3a54eb553ed4fd. Commit ID could change, though.

I think you should update your master HEAD to latest master HEAD of this repo and then apply those commits.

Let me know it you need support for it.

**Comment by brendenowens** [comment]
Hopefully, it's fixed now.

**Comment by eguzki** [comment]
Good job!

Will be released on v0.1.7 as soon as version bumping is accepted https://github.com/3scale/3scale-api-ruby/pull/44

**Pull request merged**